### ISSUE TITLE
display decimals in numeric columns

### ISSUE THREAD

**Issue by bradleyjeck** [jrowen/rhandsontable#213]
Hi, This is a great package.  Thanks for all the effort on it. 

How are the number of decimals shown in numeric columns decided?  When I load this page https://jrowen.github.io/rhandsontable/  the numeric column appears as -1,0,1 even though the underlying values are decimals. 

In my own app I have a table with some probabilities and these are getting displayed as 0 or 1 even though the decimal itself is stored. 

Thanks in advance for any insight.

**Comment by helgasoft** [comment]
It seems to be a bug in v.3.5.1
```
rhandsontable(mtcars[8:10,1:3])
```
![rt_bug](https://user-images.githubusercontent.com/13038071/34958127-4f872252-f9e5-11e7-9ed7-7e960f816eb5.png)

**Comment by kamis89** [comment]
Hi,
The same problem is with other formatting. For example doesn't work argument format = "0.0%". It's work on previous version 3.4.

I'm looking forward for solution of this problem.

With regards,
KS

**Comment by jrowen** [comment]
This should be fixed in the dev version, and we'll try to get an update on CRAN soon.
```
devtools::install_github("jrowen/rhandsontable")
```

**Comment by jrowen** [comment]
Fixed in v0.3.6.  Thanks for reporting.
